synapse/lib/lmdbslab.py

```diff
@@ -941,6 +945,7 @@ def _growMapSize(self, size=None):
         self.mapsize = mapsize
 
         self.resizeevent.set()
+        [callback() for callback in self.resizecallbacks]
```

Comment:
Do we want to do anything about these callbacks possible throwing an exception? Feels like a bad place to break down from a random callback.

Reply:
Can we make the above logger.warning() call a logger.info call while we are changing this code? That warning level is an artifact of the original lmdb implementation and is not an indicator of something amiss.

Reply:
Yeah good point, will catch and log exceptions here